product_docs/docs/pem/9/monitoring_performance/notifications.mdx

```diff
@@ -130,11 +134,35 @@ Use the **Payload** tab to define the JSON data to send to the endpoint when an
 
 -   Select **Test Connection** to test notification delivery to the mentioned endpoint.
 
+### Defining webhook alert levels
+
 Use the **Notifications** tab to specify an alert level for webhook endpoints:
 
 -   Set **All alerts** to **Yes** to enable all alert levels to send notifications.
 -   To send a notification when a specific alert level is reached, set the slider next to an alert level to **Yes**. You must set **All alerts** to **No** to configure an individual alert level.
 
+### Example of configuring a webhook using Slack
+
+Slack provides a way to post messages via `Incoming Webhooks`. To send an alert from PEM to Slack, users must create an `Incoming Webook`. Refer to Slack's guide [Sending messages using incoming webhooks](https://api.slack.com/messaging/webhooks). The `Incoming Webhook` gives a unique URL to which the pemAgent can send a JSON payload with the message text and other options to Slack webhooks.
+
+Here are the steps to configure a Slack `Incoming Webhook` using PEM:
+
+- Configure the Slack application, if it's not already configured.
+- Activate `Incoming Webhooks` from Slack.
+- Add new webhooks to the Slack workspace. You must add the channel and authorize the application for the incoming webhooks so the pemAgent can notify the alert using Slack.
+- Get the unique URL used in PEM to configure the webhook configurations. You can configure PEM to send notifications (low, medium, high) to Slack via a POST request.
+- The pemAgent uses the URL to send alert notifications from PEM to Slack using an HTTP POST request, as configured in the webhook.  
```

Comment:
I think this line is trying to say that the user should enter the unique URL generated by Slack into the Webhook configuration in PEM (as described above), if so we need to correct the wording. In any case this whole section needs much more explicit instructions.